### PR TITLE
fix light orientation

### PR DESCRIPTION
**Pull request by screwt**
There was a bug concerning sun light export orientation was wrong.

Blender 2.78c
Ogre 2.1

### CODE REVIEW

**Comment by paroj** [comment]
this needs more checks as the exporter should still work with Ogre 1.10 and is also used by the jMonkeyEngine.

File: io_ogre/ui/export.py [hunk 1]
```diff
@@ -244,137 +257,12 @@ def execute(self, context):
         name="copy shader programs",
         description="when using script inheritance copy the source shader programs to the output path",
         default=config.get('COPY_SHADER_PROGRAMS'))
-
```
Comment (paroj) [inline io_ogre/ui/export.py]: why did you remove all the config options?

Comment (screwt) [inline io_ogre/ui/export.py]: They were duplicated, see line 128, 

https://github.com/OGRECave/blender2ogre/pull/10/files/d470ff218fa987c161fffcd773bd779d292cac3b?diff=split#diff-f1a0c18781221e75057648cf561b01c9R128

Comment (screwt) [inline io_ogre/ui/export.py]: This commit has duplicated the config values. https://github.com/OGRECave/blender2ogre/commit/914fbced9fbe9d6baa27a0b33029a7f68cc0b1bf#diff-149842062e0cc714c18be472974455ddL138

**Comment by screwt** [comment]
All right i just commited to my fork and the last commit are not part of the light problem.
Can you delete the PR?

File: io_ogre/ui/export.py (hunk 1 above)
Comment (paroj) [inline io_ogre/ui/export.py]: :+1:
